common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -15212,14 +15074,6 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=638)
     public static final String A_zimbraPrefZimletTreeOpen = "zimbraPrefZimletTreeOpen";
 
-    /**
-     * Option to turn on/off zimbra mobile gateway push notifications
-     *
-     * @since ZCS 8.7.0,9.0.0
-     */
-    @ZAttr(id=1952)
-    public static final String A_zimbraPrefZmgPushNotificationEnabled = "zimbraPrefZmgPushNotificationEnabled";
```

Comment:
if I understand correctly then these provisioning files are auto generated from zimbra-attrs.xml file, and we haven't removed ldap attributes from zimbra-attrs.xml file so any developer will regenerate these files and the code will be back

Reply:
also we usually just deprecate ldap attributes, we can't remove it as it can cause issues in upgrade paths.
I might be wrong here, backend devs will have more idea about this restriction.

Reply:
This has been addressed via https://github.com/Zimbra/zm-mailbox/pull/1570